packages/tocco-util/src/appFactory/appFactory.js

```diff
@@ -37,8 +39,14 @@ export const createApp = (name, content, reducers, sagas, input, events, actions
         setLocale: locale => intl.setLocale(store, name, locale)
       }
     }
-  } catch (e) {
-    console.log('Error creating react application: ', e)
+  } catch (error) {
+    try {
+      if (store) {
+        store.dispatch(logError('Error', 'Error creating react application: ', error))
+      }
+    } catch (logError) {
```

Comment:
- IMO this variable shouldn't be called `logError` - there is already an import called `logError`
- shouldn't we log both errors? (`error` and `logError`)